extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -50,13 +62,63 @@ public static function getSvg( $name, $classNames = '', $alt = '' ) {
 	 * @return SimpleXMLElement
 	 */
 	private static function getCachedSvg( $name ) {
-		if ( isset( self::$svgCache[ $name ] ) ) {
-			$xml = self::$svgCache[ $name ];
+		if ( isset( self::$svgCache[$name] ) ) {
+			$xml = self::$svgCache[$name];
 		} else {
 			$xml = simplexml_load_file( self::SVG_DIR . '/' . $name . '.svg' );
-			self::$svgCache[ $name ] = $xml;
+			self::$svgCache[$name] = $xml;
 		}
 
 		return $xml;
 	}
+
+	/**
+	 * @desc Renders text based on value of the `type` field, supports following types:
+	 *       - text
+	 *       - translatable-text
+	 *       - link-text
+	 *
+	 * @param array $fields
+	 *
+	 * @return string
+	 */
+	public static function renderText( $fields ) {
+		if ( $fields['type'] === 'text' ) {
+			return htmlspecialchars( $fields['value'] );
+		}
+
+		if ( $fields['type'] === 'translatable-text' ) {
```

Comment:
It doesn't change the funcionality but it made me think what should happen when no condition is met, so thanks for that :)